test_msgs/include/test_msgs/service_fixtures.hpp

```diff
@@ -136,4 +139,79 @@ get_services_basic_types()
   return services;
 }
 
+std::vector<
+  std::pair<
+    test_msgs::srv::Arrays::Request::SharedPtr,
+    test_msgs::srv::Arrays::Response::SharedPtr
+  >
+>
+get_services_arrays()
+{
+  std::vector<
+    std::pair<
+      test_msgs::srv::Arrays::Request::SharedPtr,
+      test_msgs::srv::Arrays::Response::SharedPtr
+    >
+  > services;
+  {
+    auto basic_types_msgs = get_messages_basic_types();
```

Comment:
Nitpick: maybe move it closer to where it is used the first time. When reading this I was puzzled at first what it is good for...

Same in Python below.

Reply:
2f0076c + 654a7ea